adapters/rubicon/rubicon.go

```diff
@@ -231,6 +234,19 @@ func Builder(bidderName openrtb_ext.BidderName, config config.Adapter, server co
 }
 
 func (a *RubiconAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	requestCopy, err := copystructure.Copy(request)
+	if err != nil {
+		return nil, []error{err}
+	}
```

Comment:
`copy` method used is not preferred here. How about simply assigning request to new var and using it

```
requestCopy := request
```